server/chat-commands/info.ts

```diff
@@ -2509,14 +2509,14 @@ export const commands: ChatCommands = {
 				return this.errorReply('Invalid image');
 			}
 		}
-		buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray"><small>(Requested by ${request.name})</small>`;
+		buf += Utils.html`<br /><div class="infobox"><small>(Requested by ${request.name})</small>`;
 		if (request.comment) {
-			buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray">${request.comment}</p>`;
+			buf += Utils.html`<br />${request.comment}</small></div>`;
 		} else {
-			buf += `</small></p>`;
+			buf += `</small></div>>`;
 		}
-		this.addBox(buf);
-		room.update();
+		room.add(`|c|${request.name}|/html ${request.name},${buf}`);
```

Comment:
Have you tested this? I'm pretty sure this isn't what you wanted. I'm also pretty sure I already told you the correct way to do this:

https://github.com/smogon/pokemon-showdown/pull/6907#discussion_r444684882

And once again, you marked it as resolved without actually fixing it.